### ISSUE TITLE
[Mechanic] Calendar and Events!

### ISSUE THREAD

**Issue by Frostwhiff** [dalaranwow/dalaran-wow#5415]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by teste
rs and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: There are no events on the Calendar exept the Fishing contests that is somewhat anoying for me, because I like taking part in the weekly and anual events as well as the Darkmoonfair, which is not on the Calendar at all. Why?
I get the impression the Server is dying out, problems dont seem to get fixed that is very sad!
**Expected behaviour**: 

**Steps to reproduce the problem**:

1. open Calendar
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

![calendar 1](https://user-images.githubusercontent.com/38619731/39095719-3ddbc042-4645-11e8-82d4-8cc2f0ebf491.jpg)
![calendar2](https://user-images.githubusercontent.com/38619731/39095787-8ef8e184-4646-11e8-8029-76464cc9cc9c.jpg)

**Comment by torindel** [comment]
Calendar events are highwire too
New char, on new account, without guild, Level 1 and i got some random guild calendar events:
![wowscrnshot_060118_194222](https://user-images.githubusercontent.com/15069702/40855167-0df46290-65d4-11e8-8417-cebba2c7fc99.jpg)

**Comment by Lothloryen** [comment]
This is now live, thank you.
